be/src/olap/memory/write_tx.h

```diff
@@ -0,0 +1,58 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#pragma once
+
+#include "olap/memory/common.h"
+#include "olap/memory/partial_row_batch.h"
+#include "olap/memory/schema.h"
+
+namespace doris {
+namespace memory {
+
+class PartialRowBatch;
+
+// Class for write transaction
+//
+// Note: Currently it stores all its operations in memory, to make things simple,
+// so we can quickly complete the whole create/read/write pipeline. The data structure may
+// change as the project evolves.
+//
+// TODO: add write to/load from WritexTx files in future.
+class WriteTx {
```

Comment:
The abbreviation of transaction is Txn. So better rename to `WriteTxn`

Reply:
Both are valid abbreviations.
![img](https://www.allacronyms.com/4871909rbot.png)

Reply:
Alright. However, other parts of the code use `txn` as an abbreviation, so it is recommended to unify.

Reply:
OK